.github/workflows/PR_info_check.yml

```diff
@@ -6,12 +6,33 @@ on:
       - edited
       - reopened
       - synchronize
+    branches:
+      - develop
+      - master
 
 jobs:
   check-commit-message:
     name: Check Commit Message
     runs-on: ubuntu-latest
     steps:
+      - name: Check PR JIRA reference
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(\[WIP\] )?((EPMLSTRCMW-\d{1,4})|(IS-00)) .*$'
```

Comment:
Shouldn't we use a commit type here also?

Reply:
Currently we don't have such requirement anywhere.
We can discuss it, update documentation and then add such check. But I don't think it's necessary because, as the name implies, it's a _commit_ type :) PR's name doesn't have to strictly match commit message and PR might even have several commits with different types.